qcfractal/interface/collections/collection.py

```diff
@@ -238,6 +238,10 @@ def save(self, client: Optional['FractalClient'] = None) -> 'ObjectId':
 
         self._pre_save_prep(client)
 
+        if self.data.owner is None:
+            if self.client.username is not None:
+                self.data.__dict__["owner"] = self.client.username
```

Comment:
Pretty sure we don't want to do this until we have fleshed out what `owner` really means?

Reply:
The idea was to make this work:
```
client = ptl.FractalClient(**login)
ds = ptl.collections.Dataset(name="foo", client=client)
ds.save()
client.list_collections()
```